packages/paste-core/components/spinner/src/styled.tsx

```diff
@@ -0,0 +1,27 @@
+import {styled} from '@twilio-paste/styling-library';
+
+import {SvgKeyframes, CircleKeyframes} from './keyframes';
+import {strokeDasharray} from './constants';
+
+export const StyledCircleTrack = styled.circle(() => {
```

Comment:
nit: since you don't need any props, this can be simplified to
```
styled.circle({
  transformOrigin: 'center',
  opacity: 0.25,
})
```
same on line 22 (reduces by 1 function call each)